timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -33,7 +41,25 @@ types:
           immutableTimestamp: LockImmutableTimestampResponse
           timestamps: PartitionedTimestamps
           lease: Lease
-
+      StartTransactionsWithWatchesRequest:
+        fields:
+          requestId: uuid
+          requestorId: uuid
+          numTransactions: integer
+          lastKnownVersion: optional<safelong>
+        docs: |
+          An empty lastKnownVersion is treated as a request for a snapshot update. If the list of all events since last
+          known version is not available anymore, we will attempt to return a snapshot update.
+      StartTransactionsWithWatchesResponse:
+        fields:
+          immutableTimestamp: LockImmutableTimestampResponse
+          timestamps: PartitionedTimestamps
+          lease: Lease
+          lockWatchUpdate: LockWatchStateUpdate
+      GetCommitTimestampResponse:
+        fields:
+          commitTimestamp: safelong
```

Comment:
`safelong` actually only goes up to 2^53, so I don't think we should use it for (AtlasDB) timestamps.

Reply:
There is a world where we decide that it's fine to limit timestamps to 2^53, but I wanted to check that we decided where that is not OK